internal/cli/commands_test.go

```diff
@@ -1,16 +1,20 @@
 package cli
 
 import (
+	"errors"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strconv"
+	"strings"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/suite"
 )
 
+var noErrorExpected error = nil
```

Comment:
There's no value in this variable. Just use `nil` directly for `expectedErr` in the tests

Reply:
fixed in r3code@f20abab